packages/client/components/ReflectionCard/ReactjiCount.tsx

```diff
@@ -51,15 +54,38 @@ const Count = styled('div')({
   paddingLeft: 4
 })
 
+export const SHOW_REACTJI_USERS_DELAY = 100 // ms
```

Comment:
+1 could add this to `constEnums` but I don't think it matters too much https://github.com/ParabolInc/parabol/blob/26cb10c809739921f3c65e04d36ddd6e238585fb/packages/client/types/constEnums.ts#L398

Reply:
moved const to `Times`, although I think it's better when the definition is as close as possible to the place of usage. not a big deal, let's stick to the current pattern!